metadata-ingestion/src/datahub/ingestion/source/grafana/models.py

```diff
@@ -276,12 +275,13 @@ def extract_dashboard_data(cls, data: Any) -> Dict[str, Any]:
             else:
                 panels = _panel_data
 
-            meta = dashboard_data.get("meta", {})
-            folder_id = meta.get("folderId") if meta else None
-
             result = {**dashboard_data, "panels": panels}
-            if folder_id is not None:
-                result["folder_id"] = folder_id
+
+            if (meta := data.get("meta")) is not None:
+                # We only want to set folder_id and created_by from meta if we get it
+                # from json and not when creating Dashboard(uid=...) like in tests.
```

Comment:
```suggestion
                # "meta" is root field in the json
                # https://grafana.com/docs/grafana/latest/developer-resources/api-reference/http-api/dashboard/#get-dashboard-by-uid
```

> "not when creating Dashboard(uid=...) like in tests"? 
Do you mean this is wrong?
https://github.com/datahub-project/datahub/blob/3e00d3f655caa3a11d365596dd99072844840748/metadata-ingestion/tests/unit/grafana/test_grafana_entity_mcp_builder.py#L34-L46